saleor/graphql/order/schema.py

```diff
@@ -108,7 +108,7 @@ class OrderQueries(graphene.ObjectType):
     )
     order_by_token = graphene.Field(
         Order,
-        description="Look up an order by token.",
+        description="Look up an order by token. {DEPRECATED_IN_3X_FIELD}",
```

Comment:
IMHO `DEPRECATED_IN_3X_FIELD` should be placed at the beggining of description. (It is done in that way for the rest of depreceated descriptions)

Reply:
Will be changed in #9295